tests/integration/golang/mlflow/run/create_test.go

```diff
@@ -105,17 +199,31 @@ func (s *CreateRunTestSuite) Test_Ok() {
 }
 
 func (s *CreateRunTestSuite) Test_Error() {
-	_, err := s.NamespaceFixtures.CreateNamespace(context.Background(), &models.Namespace{
+	namespace, err := s.NamespaceFixtures.CreateNamespace(context.Background(), &models.Namespace{
 		ID:                  1,
 		Code:                "default",
 		DefaultExperimentID: common.GetPointer(int32(0)),
 	})
 	assert.Nil(s.T(), err)
 
+	experiment, err := s.ExperimentFixtures.CreateExperiment(context.Background(), &models.Experiment{
+		Name:           uuid.New().String(),
+		NamespaceID:    namespace.ID,
+		LifecycleStage: models.LifecycleStageActive,
+	})
+	assert.Nil(s.T(), err)
+
+	// set namespace default experiment.
+	namespace.DefaultExperimentID = experiment.ID
+	_, err = s.NamespaceFixtures.UpdateNamespace(context.Background(), namespace)
+	assert.Nil(s.T(), err)
+	nonExistingExperimentID := *experiment.ID + 1
```

Comment:
this could really confuse -> `+1`. I mean why not just put anything into `nonExistingExperimentID` without such a strange calculation? just `-1` or `99999` or anything else?

Reply:
well this is ensuring that whatever ID was just assigned for the existing experiment could not possibly collide with our non-existent id. I think PG tests are using sequence for ID which continues to increase on every run even when we truncate tables.